src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -119,30 +125,44 @@ const SetAclPermsDocContainerForm = () => {
       statusType="Permission status"
       defaultMessage="To be set..."
     >
-      <form onSubmit={handleAclPermission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-to">Set permissions to username: </label>
-          <br />
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleAclPermission} autoComplete="off">
+          <FormControl>
+            <Typography htmlFor="set-acl-to">Set permissions to username:</Typography>
+            <TextField
+              id="set-acl-to"
+              name="setAclTo"
+              {...username}
+              placeholder={selectedUser}
+              label="Search username"
+              required
+            />
+          </FormControl>
           <br />
-          <input
-            id="set-acl-to"
-            size="25"
-            name="setAclTo"
-            {...username}
-            placeholder={selectedUser}
-          />
-        </div>
-        <div style={formRowStyle}>
-          <p>Select permission setting:</p>
-          <input type="radio" id="set-acl-perm-give" name="setAclPerms" value="Give" />
-          <label htmlFor="set-acl-perm-give">Give</label>
-          <input type="radio" id="set-acl-perm-revoke" name="setAclPerms" value="Revoke" />
-          <label htmlFor="set-acl-perm-revoke">Revoke</label>
-        </div>
-        <button disabled={state.processing} type="submit">
-          Set Permission
-        </button>
-      </form>
+          <FormControl fullWidth>
+            <FormLabel id="set-acl-perm-label">Select permission setting:</FormLabel>
+            <RadioGroup row aria-labelledby="set-acl-perm-label" name="set-acl-perm">
```

Comment:
The radio group is off centered, but it can be fixed with a sx attribute: `sx={{ display: 'flex', justifyContent: 'center' }}`. It's pretty small, so I can do it on my end.

Other than that, think it's ready for merging.